release/reorder-tags.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# for some reason, github always puts our enterprise release notes above our open source
```

Comment:
Maybe we can link this thread here: https://github.com/orgs/community/discussions/8226